### PR TITLE
[SCAFCLOUD-572] Add additional logging for debugging 

### PR DESCRIPTION
**Pull request by tard916**

### CODE REVIEW

File: sonarqube-qa-scan/index.js [hunk 1]
```diff
@@ -18,24 +18,36 @@ const createAnalysisComment = (qualityGateConditions, branchName) => {
...
+  try {
+    console.log(`PullNumber ${pullNumber}`)
+    const octokit = github.getOctokit(token);
+    await octokit.rest.issues.createComment({
+      owner: 'servicerocket',
+      repo: repoName,
+      issue_number: pullNumber,
+      body: comment
+    });
+  }catch (e) {
```
Comment (brigitte0) [inline sonarqube-qa-scan/index.js:30]:
```suggestion
  } catch (e) {
```

File: sonarqube-qa-scan/index.js [hunk 2]
```diff
@@ -18,24 +18,36 @@ const createAnalysisComment = (qualityGateConditions, branchName) => {
...
 const getBranchAnalysis = async (sonarQubeProjectKey, apiToken, branchName) => {
-  const SONAR_HOST = "https://t-qa.performancerocket.com";
-  const url = `${SONAR_HOST}/api/qualitygates/project_status?projectKey=${sonarQubeProjectKey}&branch=${branchName}`;
-  const options = {
-    method: 'GET',
-    headers: { Authorization: `Basic ${apiToken}`, 'Content-Type': 'application/json' },
-    url,
-  };
-  return axios(options);
+  try{
```
Comment (brigitte0) [inline sonarqube-qa-scan/index.js:37]:
```suggestion
  try {
```

File: sonarqube-qa-scan/index.js [hunk 3]
```diff
@@ -18,24 +18,36 @@ const createAnalysisComment = (qualityGateConditions, branchName) => {
...
+    const SONAR_HOST = "https://t-qa.performancerocket.com";
+    const url = `${SONAR_HOST}/api/qualitygates/project_status?projectKey=${sonarQubeProjectKey}&branch=${branchName}`;
+    console.log(`Quality gate URL ${url}`)
+    const options = {
+      method: 'GET',
+      headers: { Authorization: `Basic ${apiToken}`, 'Content-Type': 'application/json' },
+      url,
+    };
+    return axios(options);
+  }catch (e) {
```
Comment (brigitte0) [inline sonarqube-qa-scan/index.js:47]:
```suggestion
  } catch (e) {
```

**Review by brigitte0 — APPROVED** [review]
Consistent spacing

**Review by yoonkean — APPROVED** [review]

**Pull request merged**